### PR TITLE
Moving nest hack

### LINKED ISSUE #1521: WRF_v4.3 out-of-the-box doesn't compile in vortex-following mode
**Describe the bug**
The vortex-following compilation mode doesn't compile using Intel 2019_u3
[Intelu3_configure_wrf.txt](https://github.com/wrf-model/WRF/files/6525773/Intelu3_configure_wrf.txt)
[Intelu3_log_cmp_153.txt](https://github.com/wrf-model/WRF/files/6525774/Intelu3_log_cmp_153.txt)

**To Reproduce**
1. Use compiler and version: 'Intel, 2019.3.199' (see the attached `configure.wrf`)
2. Download WRF_v4.3 out-of-the-box code 
3. Output is: see the attached output `log_cmp_153`

**Expected behavior**
The code is expected to compile using option: 15 (Intel dampar) + 3 (vortex following)

**Screenshots**
See the attached output log 

**Attachments**
See the attached output log 

**Additional context**

### PR DESCRIPTION
**Pull request by davegill**
TYPE: bug fix
    
KEYWORDS: moving nest, vortex tracking, segfault, Intel
    
SOURCE: internal
    
DESCRIPTION OF CHANGES:
Problem:
When the Intel compiler is used with the moving nest option (vortex tracking), the tools/registry program seg-faults,
and does not complete building the automatically generated files that are included via cpp into the WRF source code.
    
Solution:
This is a quick-fix for users. Simply swap the order of the list of registry files.
    
ISSUE: For use when this PR closes an issue.
Fixes #1521
    
LIST OF MODIFIED FILES:
M   Registry/registry.em_shared_collection
    
TESTS CONDUCTED:
1. This mod allows the code to build successfully for intel 19.0.5
2. The code compiles with option 15/3 using Intel OneAPI:  `ifort (IFORT) 2021.1 Beta 20201112`
3. The code compiles with option 15/3 and run using Intel HPC suit: `ifort (IFORT) 19.0.5.281 20190815`. The code 
runs at least 1-h with a triple-nests 501 x 601 x 841 Tropical Cyclone with 55 levels.
4. Jenkins is all pass.

RELEASE NOTES: A temporary fix has been put in place for the moving nest capability when using the Intel compiler. Previously, the WRF source code did not build due to a segmentation fault in the execution of the tools/registry program.

### CODE REVIEW

**Comment by davegill** [comment]
@dudhia @weiwangncar @JS-WRF-SBM 
Folks,
This is a temporary fix

**Comment by davegill** [comment]
jenkins is OK

```
Please find result of the WRF regression test cases in the attachment. This build is for Commit ID: 7a42396052f9cc5b071432b64b790678983859ec, requested by: davegill for PR: https://github.com/wrf-model/WRF/pull/1555. For any query please send e-mail to David Gill.

    Test Type              | Expected  | Received |  Failed
    = = = = = = = = = = = = = = = = = = = = = = = =  = = = =
    Number of Tests        : 19           18
    Number of Builds       : 48           46
    Number of Simulations  : 163           161        0
    Number of Comparisons  : 103           102        0

    Failed Simulations are: 
    None
    Which comparisons are not bit-for-bit: 
    None
```

**Comment by weiwangncar** [comment]
@davegill Interesting fix...

**Comment by weiwangncar** [comment]
@davegill Looks ok to me...

**Comment by JS-WRF-SBM** [comment]
@davegill  Very much appreciated!
I guess I'll wait until it is merged to the `dev` repo.
Best,
Koby

**Comment by davegill** [comment]
@JS-WRF-SBM 
Would you clone this repo, checkout the branch, and see if this code works for you?

**Comment by JS-WRF-SBM** [comment]
> @JS-WRF-SBM
> Would you clone this repo, checkout the branch, and see if this code works for you?

@davegill 
Sure thing, give me a couple of days to get back to you.

**Review by smileMchen — APPROVED** [review]
I have compiled the code successfully.

**Comment by JS-WRF-SBM** [comment]
@davegill 

The code compiles 15/3 with (Laptop):
- Intel OneAPI:  `ifort (IFORT) 2021.1 Beta 20201112`

The code compiles 15/3 and run (HPC):
- Intel HPC suit: ifort (IFORT) 19.0.5.281 20190815
The code runs at least 1-h with a triple-nests 501 x 601 x 841 Tropical Cyclone with 55 levels.

Thanks for your work on this!

**Comment by davegill** [comment]
@JS-WRF-SBM 
THANK YOU FOR YOUR TESTING!

**Review by kkeene44 — APPROVED** [review]

**Pull request merged**
